traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -1803,15 +1811,11 @@ sub regex_revalidate_dot_config {
 		}
 		my $asset_url = $row->asset_url;
 
-		my $job_cdn_id = $row->job_deliveryservice->cdn_id;
-		if ( $cdn_obj->id == $job_cdn_id ) {
-
-			# if there are multipe with same re, pick the longes lasting.
-			if ( !defined( $regex_time{ $row->asset_url } )
-				|| ( defined( $regex_time{ $row->asset_url } ) && $purge_end > $regex_time{ $row->asset_url } ) )
-			{
-				$regex_time{ $row->asset_url } = $purge_end;
-			}
+		# if there are multipe with same re, pick the longes lasting.
```

Comment:
minor nit -- 2 misspellings

Reply:
done

Reply:
Yeah, I saw the spellings. They were there in the original before it was unindented. Probably an error in translation from the original Dutch. 👀